style.css

```diff
@@ -5,11 +5,16 @@
   list-style: none;
 }
 
+body {
+  background-color: #e5e5e5;
+}
+
 .headline-bar {
   height: 72px;
   display: flex;
   justify-content: space-between;
   align-items: center;
+  background-color: white;
 }
```

Comment:
- [ ] Kindly adjust the margin-top from the headline section to the work section to match the original design. Also adjust the font family. 

![image](https://user-images.githubusercontent.com/98453979/205040382-fc3805bd-ef4d-45b6-a20a-1f0ea9c5d67e.png)

- [ ] Kindly add the  missing grey dots in this section to match the original design
  
![image](https://user-images.githubusercontent.com/98453979/205041079-c7038d94-11dd-4deb-b3d3-2362beed7081.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/205041188-35996bfc-0bcb-4b75-b09b-0606ab1b16a7.png)

- [ ] Kindly center your project cards and increase the margin from both sides to match the original design

![image](https://user-images.githubusercontent.com/98453979/205042159-76e54381-1bd4-4dac-b16e-247ea9b72d78.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/205042310-cb631679-9582-4cf6-aa23-4f16272aadbc.png)

- [ ] Kindly position the `get my resume`  button in the `about section` correctly to align well with  other elements to match the original design

![image](https://user-images.githubusercontent.com/98453979/205042897-ee5de0e8-1f91-49bd-b277-e845394d80be.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/205043010-5247d7c8-6447-4b8e-9f50-022b52ad09c8.png)